### DISCUSSION TITLE
Budgie 10.7.1

### DISCUSSION THREAD

**Discussion by JoshStrobl** [BuddiesOfBudgie/budgie-desktop#310] [Announcements]
Budgie 10.7.1 is the first minor release in the 10.7 series of our Budgie Desktop environment, introducing some improvements, bug fixes, and updated translations.

Learn more at https://blog.buddiesofbudgie.org/budgie-10-7-1-released/

<hr /><em>This discussion was created from the release <a href='https://github.com/BuddiesOfBudgie/budgie-desktop/releases/tag/v10.7.1'>Budgie 10.7.1</a>.</em>

**Comment by OlivierDuchateau** [comment]
Everything works fine on FreeBSD. Thanks!

**Comment by Hi-Phile** [comment]
I'm not sure what happened, but now Raven doesn't seem to work, it's not showing/appearing on the taskbar.
![image](https://user-images.githubusercontent.com/35350630/224188567-de50ac0a-1a16-4206-8d89-bf352ae73ff9.png)
![image](https://user-images.githubusercontent.com/35350630/224188709-294a9762-9f75-4dcf-b4ba-debdd14755e7.png)

**Reply by EbonJaeger** [reply]
There doesn't appear to be a Raven trigger on your panel. You should be able to add it by clicking Top Panel in the menu on the left and adding the Raven Trigger Applet. No idea why it would have disappeared, though...

**Reply by Hi-Phile** [reply]
Not sure what happened either, on another system the Raven panel appears.  I don't have anything appearing when left-clicking on the Top Panel.

I even tried the budgie-desktop-git from AUR and noticed it's changed a little but Raven still doesn't appear.

Is there anywhere I can look for Budgie/Raven logs?

**Reply by Hi-Phile** [reply]
Thank you @EbonJaeger.  It turned out that Raven Trigger Applet was missing, in fact, there was no 'End' section, and I don't know why.  At first I didn't understand Top Panel, and thus my reply  comment is incorrect.  It's been corrected, but I totally missed and didn't understand that what you had described, and I should have re-read your comment.  It's sorted now, thank you.

![image](https://user-images.githubusercontent.com/35350630/224415343-d8373b46-3401-4d91-8f91-d84bccc9c249.png)

**Comment by melomane63** [comment]
hello, in ubuntu 23.04 and budgie 1.7.1 have this problem:
![Screenshot from 2023-03-29 16-45-54](https://user-images.githubusercontent.com/44552926/228581824-4856479c-e01f-4406-af47-5bf482028dca.png)

i have only 2 workspace, and click on workspace left right ans up not work.
